src/DefaultTheme.res

```diff
@@ -55,6 +55,7 @@ let defaultRules = theme =>
     },
     ".Label": {
       "color": theme.colorText,
+      "opacity": "10",
```

Comment:
Is this change not required for other themes?

Reply:
it should be ... but for default it was looking really vibrant hence only changed default theme